cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
```

Comment:
Do you think we can get rid of these files?

Reply:
Get em out of here